sections/dashboard/EligibilityBox/EligibilityBox.tsx

```diff
@@ -0,0 +1,82 @@
+import { useCallback, useState } from 'react';
+import styled from 'styled-components';
+
+import Button from 'components/Button';
+import Input from 'components/Input/Input';
+import Text from 'components/Text';
+
+import EligibilityMap from './EligibilityMap';
+
+const EligibilityBox = () => {
+	const [address, setAddress] = useState('');
+	const [isEligible, setIsEligible] = useState<boolean>();
+
+	const handleChangeAddress = useCallback((e) => {
+		setAddress(e.target.value);
+	}, []);
+
+	const handleCheck = useCallback(() => {
+		setIsEligible(!!(!!address && EligibilityMap[address]));
```

Comment:
Could remove the !! before address as have it on the wrapper anyway

Reply:
Changed.